src/validation.cpp

```diff
@@ -3291,6 +3311,91 @@ void CChainState::ProcessGovEvents(const CBlockIndex* pindex, CCustomCSView& cac
     cache.EraseStoredVariables(static_cast<uint32_t>(pindex->nHeight));
 }
 
+void CChainState::ProcessProposalEvents(const CBlockIndex* pindex, CCustomCSView& cache, const CChainParams& chainparams) {
+    if (pindex->nHeight < chainparams.GetConsensus().GreatWorldHeight) {
+        return;
+    }
+
+    if (pindex->nHeight == chainparams.GetConsensus().GreatWorldHeight) {
+        auto balance = cache.GetBalance(chainparams.GetConsensus().foundationShareScript, DCT_ID{0});
+        if (balance.nValue > 0) {
+            cache.SubBalance(chainparams.GetConsensus().foundationShareScript, balance);
+            cache.AddCommunityBalance(CommunityAccountType::CommunityDevFunds, balance.nValue);
+        }
+    }
```

Comment:
I will gonna to remove that check, whether foundation receives tokens they will be transferred to `Community Dev Funds`

Reply:
yup, sounds good.